rust/ql/lib/codeql/rust/internal/PathResolution.qll

```diff
@@ -1773,6 +1773,8 @@ private module Debug {
     path = p.toStringDebug()
   }
 
+  predicate debugItemNode(ItemNode item) { item = getRelevantLocatable() }
```

Comment:
Actually, I think it is easier just to make `getRelevantLocatable` public (I already do that in on of my PRs).

Reply:
Yes; perhaps your use case was different from mine; I would add calls to `Debug::getRelevantLocatable` throughout the source code when debugging.